client/vscode/package.json

```diff
@@ -260,29 +260,29 @@
   },
   "scripts": {
     "lint:js": "pnpm eslint --cache '**/*.[jt]s?(x)'",
-    "test": "yarn run -T ts-node ./tests/runTests.ts",
-    "package": "yarn run -T ts-node ./scripts/package.ts",
-    "prebuild": "yarn build-inline-extensions",
-    "prewatch": "yarn build-inline-extensions",
+    "test": "ts-node ./tests/runTests.ts",
```

Comment:
This doesn't need the `pnpm` equivalent of `-T`? I think it's `-W`.

Reply:
Where are you finding the docs for these flags? I'm having trouble finding any info on `yarn -T` or `pnpm -W`...

Reply:
Oh, here are the docs for `-C` and `-w`.
https://pnpm.io/pnpm-cli#-c-path---dir-path

Reply:
Ok, so `-W` is equivalent to `-C ../../` (when run within `client/*/.`).

So `yarn run -T` does the same as `pnpm -W`? I can't find docs for any of this :/

Reply:
FWIW this specific line fails with yarn and pnpm since `tests/runTests.ts` doesn't exist lol

Reply:
Also FWIW I greatly prefer not having `yarn` or `pnpm` on this line. If you have neither then it will just run based on the `node_modules` folder and doesn't matter which cli tool actually read the package.json... so when switch yarn=>pnpm it wouldn't require any changes to this file if "yarn" wasn't here...

Reply:
`tests/runTests.ts` doesn't even exist anywhere in the repo so I think this specific line is broken. So looking at the line above instead:
```
"package": "yarn run -T ts-node ./scripts/package.ts",
```

The "scripts/package.ts" is at `client/vscode/scripts/package.ts`. So we want this run with the cwd being `client/vscode`, where this package.json is. So `pnpm run -w` (running from the root directory) doesn't work and seems incorrect. So what does `yarn run -T` do? lol

Reply:
:worried: